server/modifiers.py

```diff
@@ -203,3 +204,98 @@ def insert_empty_statements(self) -> Self:
         self.modified_count += 1
 
         return self
+
+    def reverse_booleans(self) -> Self:
+        """A code modifier that messes up some conditions.
+
+        This will reverse True and False keywords.
+
+        Returns:
+            The modifier instance.
+        """
+        number_boolean_pairs = []
+        for num, line in enumerate(self.file_contents):
+            if any(key in line for key in ["True", "False"]):
+                number_boolean_pairs.extend([(num, key) for key in ["True", "False"] if key in line])
```

Comment:
Should be `num in line`

Reply:
where? Everything worked

Reply:
> where? Everything worked

Sorry, I thought it was another loop at the end. I've been looking at code too much today :p